test/deprecated/IO/writersReturningBool.chpl

```diff
@@ -0,0 +1,15 @@
+use IO;
+use IO.FormattedIO;
+import FileSystem.remove;
+
+var w = openwriter("./wrb.txt");
+
+w.write("a");
+w.writeln("b");
+w.writebits(2, 2);
+w.writeBytes(0xF, 1);
+w.writef("%i", 5);
+
+w.close(); remove("./wrb.txt");
```

Comment:
It's unlikely to be a problem, but I think we probably should verify these still perform their operations appropriately, so would rather we check the contents of the `.txt` file than just delete it.  You can do that with a `.catfiles` for the test, or you can read it in and check you got what you expected